src/main/java/org/openpnp/machine/reference/feeder/ReferenceLoosePartFeeder.java

```diff
@@ -89,7 +89,12 @@ private Location getPickLocation(Camera camera, Nozzle nozzle) throws Exception
         Location location = VisionUtils.getPixelLocation(camera, result.center.x, result.center.y);
         // Get the result's Location
         // Update the location with the result's rotation
-        location = location.derive(null, null, null, result.angle);
+    	double angleCorrection=0;
+    	if(result.size.width < result.size.height) {
+    		angleCorrection = 90;
+    	}
+
+        location = location.derive(null, null, null, -(result.angle+angleCorrection + location.getRotation()));
```

Comment:
Hi @KarlZeilhofer, can you explain why you are negating the angle here? I refer to the `-` before `(result.angle...`

Reply:
Good question, which seems I cannot explain. I tried it without the negation before, but it didn't work.
I'll try to come back with a proper explanation...

Reply:
I figured it out. It actually fixes a bug that wasn't previously know. The angle sign is inverted between OpenCV and OpenPnP, so it needs to be inverted. Most of the other vision operations already do this.

Reply:
Thank you, @vonnieda !
Still I wonder why this hasn't noticed anyone before. Wasn't the ReferenceLoosePartFeeder not properly tested on a real machine?

Reply:
@KarlZeilhofer This code was released unfinished. People wanted to use it and modify it, so I merged it. That's why there are big red letters that tell you to use at your own risk in the wizard :)